ctgan/synthesizer.py

```diff
@@ -32,10 +32,13 @@ class CTGANSynthesizer(object):
             Wheight Decay for the Adam Optimizer. Defaults to 1e-6.
         batch_size (int):
             Number of data samples to process in each step.
+        n_discriminator (int):
+            Number of discriminator updates to do for each generator update.
+            Defaults to 5.
```

Comment:
I'm guessing you meant ```Defaults to 1```. Everything else looks good to me :thumbsup:

Reply:
Oh, correct. Forgot to change that one.

Reply:
I added the WGAN default value in the docstring and the actual default in the normal place.